i18n/locales/source/blockly-mooc/common.json

```diff
@@ -1347,27 +1347,8 @@
-  "specialAnnouncementHeadingHoc2018": "What will you create?",
-  "specialAnnouncementDescriptionHoc2018": "This year, the Hour of Code is all about creativity. Sign up today and receive more information about robotics giveaways from our partners and new posters for your classroom!",
-  "specialAnnouncementHeadingHoc2018Actual": "The Hour of Code is here",
-  "specialAnnouncementDescriptionHoc2018Actual": "Try our top activity for 2019: Dance Party! Featuring Katy Perry, Madonna, J. Balvin, Sia, Keith Urban, Ciara, and 25 more. And share your students’ best moves with us on social media with #HourOfCode.",
-  "specialAnnouncementDescriptionHoc2018Dance": "Sign up your class for the Hour of Code (Dec 3-9). And, try our top activity for 2019: Dance Party!  Featuring Katy Perry, Madonna, J. Balvin, Sia, Keith Urban, Ciara, and 25 more.",
-  "specialAnnouncementHeadingHoc2018Mc": "The Hour of Code is coming",
-  "specialAnnouncementDescriptionHoc2018Mc": "Sign up your class to try one of the over 150 NEW activities for the Hour of Code (Dec 3-9). Be sure to check out our brand new Minecraft activity for 2018, the Voyage Aquatic! Sign up your class today.",
-  "specialAnnouncementHeadingPostHoc2018": "Go beyond an Hour of Code",
-  "specialAnnouncementDescriptionPostHoc2018": "An hour is just the beginning. We offer comprehensive curriculum for grades K-12, professional learning programs including local workshops, and resources to help you and your school get started.",
-  "specialAnnouncementHeadingJoinProfessionalLearning2019": "Sign up for Professional Learning",
-  "specialAnnouncementDescriptionJoinProfessionalLearning2019": "98% of teachers recommend our engaging programs from elementary school to AP computer science for high school. Join our hands-on workshops with other teachers from your area. Spaces are limited, so let us know if you're interested.",
-  "specialAnnouncementDescriptionJoinProfessionalLearning2019Deadline": "Our Professional Learning Program for middle and high school teachers is OUT OF THIS WORLD! 90% of teachers recommend it and 90% of parents want their child to study computer science.",
-  "specialAnnouncementHeadingJoinProfessionalLearning2019Deadline": "Don’t miss out. Apply today!",
-  "specialAnnouncementHeadingJoinProfessionalLearning2019Trophy": "Know an Awesome Teacher?",
-  "specialAnnouncementDescriptionJoinProfessionalLearning2019Trophy": "Let them know how amazing they are by nominating them for a Professional Learning Scholarship today!",
+  "specialAnnouncementHeadingHoc2019CsForGood": "The Hour of Code is coming",
+  "specialAnnouncementDescriptionHoc2019CsForGood": "Try one of the 100 NEW activities for the Hour of Code (Dec 9-15)! This year, encourage your students to think about ways of using CS for good. Sign up your class today.",
```

Comment:
@castro-jorge this updated special announcement string is probably one that we'll want to specifically direct translators to so they don't see "lost translations"